website/src/css/_navbar.scss

```diff
@@ -22,3 +22,54 @@
   padding: 5px;
   font-weight: 300;
 }
+/*===========*/
+
+/* Fix navbar logo */
+.navbar__logo {
+  height: 26px;
+  margin: 0 10px 0 8px;
+}
+
+/* Restyle hamburger button */
+
+.navbar {
+  height: 70px;
+}
+
+.navbar__items--right {
+  position: relative;
+}
+
+.navbar__toggle {
+  width: 40px;
+  height: 40px;
+  background-color: var(--detox-color-a3);
+  border-radius: 10px;
+  justify-content: center;
+  align-items: center;
+}
+@media (max-width: 768px) {
+  .navbar__toggle {
+    position: absolute;
+    margin-right: 0;
+    top: 15px;
+    right: 25px;
+  }
+}
+
+/* Restyle search button */
+
+@media (max-width: 768px) {
+  button.DocSearch-Button {
+    background-color: var(--detox-color-a3);
+    height: 40px;
+    border-radius: 10px;
+    padding: 0 10px;
+    position: absolute;
+    top: -16px;
```

Comment:
Looks too dirty (-16px 😬 ). Try swizzling `Navbar/Search` – use wrapper strategy.
Near the index.js file create `styles.module.css` file where you can define a style overriding position to `right: 80px` (or whatever it is).

```
src/theme/Navbar/Search/styles.module.scss
src/theme/Navbar/Search/index.js
```